cloudformation/slack-router-lambda.js

```diff
@@ -0,0 +1,104 @@
+const AWS = require('aws-sdk')
+const crypto = require('crypto')
+
+const region = process.env.AWS_REGION
```

Comment:
This should also be passed as an env variable.

Reply:
Do you want me to pass it as an env variable again?

[AWS_REGION is a reserved environment variable for Lambdas](https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html) so you can just access it directly without needing to pass it as an env variable.